pgml-dashboard/Cargo.toml

```diff
@@ -24,3 +24,4 @@ bigdecimal = "0.3"
 chrono = "0.4"
 serde_json = "1"
 csv-async = "1"
+scraper = "0.14.0"
```

Comment:
It would be good to enable automatic new lines at the end of the files. For VsCode, I found [this](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files).